app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -28,6 +28,24 @@ def enrollment_analytics_attributes(enrollment, complete:)
     }
   end
 
+  def response_analytics_attributes(response)
+    {
+      fraud_suspected: response['fraudSuspected'],
+      primary_id_type: response['primaryIdType'],
+      secondary_id_type: response['secondaryIdType'],
+      failure_reason: response['failureReason'],
+      transaction_end_date_time: response['transactionEndDateTime'],
+      transaction_start_date_time: response['transactionStartDateTime'],
+      status: response['status'],
```

Comment:
IMO I think all this switching of cases might be a good hint to move the processing of the response to the USPS proofer client class, have it return a `Struct` or equivalent object that has snake_case keys already (similar to https://cm-jira.usa.gov/browse/LG-7025)